### ISSUE TITLE
vfs object issue

### ISSUE THREAD

**Issue by inquam** [gboudreau/Greyhole#158]
**Running samba: 4.3.8-ubuntu
Greyhole: 0.9.67
Symlinked file used: greyhole-samba43.so**

When using mount_shares_locally I noticed the issue after a samba upgrade.

```
[2016/05/02 12:48:25.878540,  0] ../lib/util/modules.c:48(load_module)
  Error loading module '/usr/lib/x86_64-linux-gnu/samba/vfs/greyhole.so': libsmbd-base-samba4.so: cannot open shared object file: No such file or directory
```

Checking for the file I find the following file and symlink in **/usr/lib/x86_64-linux-gnu/samba**

```
lrwxrwxrwx  1 root root      49 Aug 14  2015 libsmbd_base.so -> /usr/lib/x86_64-linux-gnu/samba/libsmbd_base.so.0
-rw-r--r--  1 root root 3327384 Apr 12 14:32 libsmbd-base.so.0
```

**Comment by inquam** [comment]
For anyone having this issue and is in need of a solution until a real fix is in place. Creating a symlink **/usr/lib/x86_64-linux-gnu/samba/libsmbd-base.so.0** to **/usr/lib/x86_64-linux-gnu/samba/libsmbd-base-samba4.so** and one from **/usr/lib/x86_64-linux-gnu/samba/libsamba-debug.so.0**  to **/usr/lib/x86_64-linux-gnu/samba/libsamba-debug-samba4.so** fixed my issue.

**Comment by gboudreau** [comment]
This is already fixed in 0.9.67, but might need you to re-install Greyhole after you Samba gets upgraded.
